spec/rubocop/cop/performance/redundant_block_call_spec.rb

```diff
@@ -46,6 +57,45 @@
     expect(cop.messages).to be_empty
   end
 
+  it "doesn't register an error when a block is passed as the only arg" do
```

Comment:
It should be "offense", not "error". Usually we shorten it down and just say "accepts a block passed as the only arg" instead of "doesn't register an offense...".